### ISSUE TITLE
[Bug]: Monterey public beta on Imac 2009 non-metal GPU accelation problems

### ISSUE THREAD

**Issue by jeremiasmanninen** [dortania/OpenCore-Legacy-Patcher#547]
### Machine Model

iMac11,1

### Application Version

Latest Release

### Application Variant

GUI (Graphical User Interface)

### What versions of macOS are you seeing the problem on?

macOS 12, Monterey

### Where does this issue happen?

Within macOS (After Logging in)

### What is the Isssue?

GPU Acceleration on Mac OS Monterey has been working as well as on Big Sur since the 0.2.5 release (2 weeks). Today, when my Mac had been on a few hours when I was browsing chrome there came black box with other colors too following my mouse. Youtube video paused for a second and I realized that when I took the mouse to the menu bar on the top the MacOS menu bar got graphical graininess as well. When I was fastly moving the windows opened whole the UI went just grazy. 

I managed to get 2 screenshots. Now using a while after a restart there seems not to be things that would make the computer unusable. ( Still same bit of graininess as I have got used to.)

### Any Additional Information

![Uploading Näyttökuva 2021-10-6 kello 15.35.39.png…]()
![Näyttökuva 2021-10-6 kello 15 37 24](https://user-images.githubusercontent.com/81366427/136207404-f03a0d06-562a-48a3-8b83-c1d90cac5bc1.png)
![Näyttökuva 2021-10-6 kello 15 43 48](https://user-images.githubusercontent.com/81366427/136207418-03fa8e6e-4eed-4738-b998-5564f8f80dde.png)

**Comment by jeremiasmanninen** [comment]
Good point the dying GPU. I have High Sierra installed on secondary partition so better checking that. Thank you!

**Comment by jeremiasmanninen** [comment]
Hello there! I tested running on High Sierra and have not realized any graphical problems with it. Would there be a chance that there is some problem with the patcher? I could provide the debug files if that kind of graphical problems can be seen on those files. Knowing that other people have also had similar problems it would be great to make sure that it is at least not the OCLP.

Adding to the previous observations I have had these graphical artifacts only when the computer goes to sleep for just 10 minutes or so. Before it only happened when Imac had been on / sleeping for hours.

Thank you for your help!

**Comment by jeremiasmanninen** [comment]
I also have had a problem with starting bar freezing and after restarting Imac starts normally.
I described it here: https://discord.com/channels/417165963327176704/835339735772823573/925668595533484062

And adding to the screenshot, it does not usually look that bad. Graphic problems on a screenshot seem to look different than what I see on the screen. Usually artifacts around the mouse and when I move it very fast I am able to locate the shutdown button for example.

**Comment by WildChild-SomethingWild** [comment]
Like what has been mentioned above by KhronoKernel, this is an AMD GPU-related problem. No fix yet. 
You could use 11.2.3 as it is the last version without that bug.

And lastly, go to server if you have questions since this issue has been marked as closed.
